CONTRIBUTING.md

```diff
@@ -21,11 +21,11 @@ We would love to get help from you as "user" and "contributor".
 The rest of this document is a guide for those maintaining Aruba, and others
 who would like to submit patches.
 
-## Contributing to the "aruba" project
+## Contributing to the `aruba` project
```

Comment:
I'd go with just writing 'Aruba' to refer to the project. As in, no quotes, and with a capital 'A'.

Reply:
Thanks, done!